jsonpb/jsonpb_test.go

```diff
@@ -821,6 +833,45 @@ func init() {
 	proto.RegisterType((*dynamicMessage)(nil), dynamicMessageName)
 }
 
+type ptrFieldMessage struct {
+	StringField *stringField `protobuf:"bytes,1,opt,name=stringField"`
+}
+
+func (m *ptrFieldMessage) Reset() {
+}
+
+func (m *ptrFieldMessage) String() string {
+	return m.StringField.StringValue
+}
+
+func (m *ptrFieldMessage) ProtoMessage() {
+}
+
+type stringField struct {
+	IsSet       bool   `protobuf:"varint,1,opt,name=isSet"`
+	StringValue string `protobuf:"bytes,2,opt,name=stringValue"`
+}
+
+func (s *stringField) Reset() {
+}
+
+func (s *stringField) String() string {
+	return s.StringValue
+}
+
+func (s *stringField) ProtoMessage() {
+}
+
+func (s *stringField) MarshalJSONPB(jm *Marshaler) ([]byte, error) {
```

Comment:
Optional.  This is not necessary since the test is only on JSONPBUnmarshaler.

Reply:
OK, removed